Merge 2.9.2 tag

gdemonet opened the pull request:
The usual (with a small docs bugfix).

NickSayer-scality committed:
Release 2.9.2

gdemonet committed:
docs: Keep including developer docs on ReadTheDocs

This was forgotten in #3351.

gdemonet committed:
Merge tag '2.9.2' into feature/merge-2.9.2-tag

MetalK8s 2.9.2
==============

MetalK8s 2.9.2 embeds Kubernetes 1.20.6.

Documentation
-------------

https://metal-k8s.readthedocs.io/en/2.9.2

Upgrade Notes
-------------

Please follow the upgrade instructions [here](https://metal-k8s.readthedocs.io/en/2.9.2/operation/upgrade.html)

Changelog since 2.9.1
---------------------

Full list of closed issues is available [here](https://github.com/scality/metalk8s/blob/2.9.2/CHANGELOG.md)

What’s new
----------

- Improved long label name in disk management in the UI
- Added consistency checks for Salt minion ID and Kubernetes Node name
- Improved ``metalk8s_network`` module

bert-e commented:
# Hello gdemonet,



My role is to assist you with the merge of this
pull request. Please type `@bert-e help` to get information
on this process, or consult the [user documentation](
https://eve.devsca.com/github/scality/metalk8s/bert-e/doc/user).





*Status report is not available.*

bert-e commented:
# Conflict




A conflict has been raised during the creation of
integration branch `w/2.10/feature/merge-2.9.2-tag` with contents from `feature/merge-2.9.2-tag`
and `development/2.10`.

**I have not created the integration branch.**


Here are the steps to resolve this conflict:

```bash
 $ git fetch
 $ git checkout -B w/2.10/feature/merge-2.9.2-tag origin/development/2.10
 $ git merge origin/feature/merge-2.9.2-tag
 $ # <intense conflict resolution>
 $ git commit
 $ git push -u origin w/2.10/feature/merge-2.9.2-tag
```

gdemonet commented:
/approve

bert-e commented:
# Waiting for approval



The following approvals are needed before I can proceed with the merge:


* the author


* one peer





Peer approvals *must* include at least 1 approval from the following list:

* @gdemonet

* @chengyanjin

* @teddyandrieux

* @jbwatenbergscality

* @alexandre-allard-scality

* @nicolast

* @thomasdanan

* @lucieleonard










*The following options are set:* **approve**

bert-e commented:
# Build failed



The build for [commit](https://github.com/scality/metalk8s/commit/8fddf975aabccad3bd272894ba82dc7059ac4fec) did not succeed in branch feature/merge-2.9.2-tag.




*The following options are set:* **approve**

ChengYanJin committed:
ui: Select node base on node label in createvolume spec

Make the test more robustness.
Without this fix, the test may fail in the node selection when the first
node `bootstrap` is the default selection.

bert-e commented:
# Waiting for approval



The following approvals are needed before I can proceed with the merge:


* the author


* one peer





Peer approvals *must* include at least 1 approval from the following list:

* @gdemonet

* @chengyanjin

* @teddyandrieux

* @jbwatenbergscality

* @alexandre-allard-scality

* @nicolast

* @thomasdanan

* @lucieleonard










*The following options are set:* **approve**

bert-e commented:
# In the queue



The changeset has received all authorizations and has been added to the
relevant queue(s). The queue(s) will be merged in the target development
branch(es) as soon as builds have passed.

The changeset will be merged in:


* :heavy_check_mark: `development/2.9`

* :heavy_check_mark: `development/2.10`



The following branches will **NOT** be impacted:

* `development/2.0`
* `development/2.1`
* `development/2.2`
* `development/2.3`
* `development/2.4`
* `development/2.5`
* `development/2.6`
* `development/2.7`
* `development/2.8`



There is no action required on your side. You will be notified here once
the changeset has been merged. In the unlikely event that the changeset
fails permanently on the queue, a member of the admin team will
contact you to help resolve the matter.

*IMPORTANT*

Please do not attempt to modify this pull request.

* Any commit you add on the source branch will trigger a new cycle after the
  current queue is merged.
* Any commit you add on one of the integration branches will be **lost**.

If you need this pull request to be removed from the queue, please contact a
member of the admin team now.




*The following options are set:* **approve**

bert-e commented:
# 


I have successfully merged the changeset of this pull request
into targetted development branches:


* :heavy_check_mark: `development/2.9`

* :heavy_check_mark: `development/2.10`



The following branches have **NOT** changed:

* `development/2.0`
* `development/2.1`
* `development/2.2`
* `development/2.3`
* `development/2.4`
* `development/2.5`
* `development/2.6`
* `development/2.7`
* `development/2.8`



Please check the status of the associated issue None.

Goodbye gdemonet.
